includes/class-event.php

```diff
@@ -0,0 +1,399 @@
+<?php
+
+namespace Automattic\WP\Cron_Control;
+
+use WP_Error;
+
+class Event {
+	private ?int $id;
+	private string $status;
+
+	// TODO: Maybe we don't need action_hashed going forward?
```

Comment:
Yes and no. We still need action_hashed since the DB table expects it, and other existing functionality uses it. Eventually once the rest of the related plugin logic is consuming this object though, it could potentially be removed.